packages/orchestration/src/orchestration-api.ts

```diff
@@ -145,17 +145,17 @@ export interface OrchestrationAccountI {
   /**
    * Transfer an amount to another account, typically on another chain.
    * The promise settles when the transfer is complete.
-   * @param amount - the amount to transfer.
+   * @param amount - the amount to transfer. Can be provided as pure data using denoms or as native Amounts.
```

Comment:
"native" is ambiguous as to whether native to Agoric or the host chain.
```suggestion
   * @param amount - the amount to transfer. Can be provided as pure data using denoms or as ERTP Amounts.
```

Reply:
👍 updating the `AmountArg` annotation as well